spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -45,10 +46,13 @@
     Signal, Slot)
 from qtpy.QtGui import QColor, QCursor
 from qtpy.QtWidgets import (
-    QApplication, QDialog, QFrame, QGridLayout, QHBoxLayout, QInputDialog,
-    QItemDelegate, QLabel, QLineEdit, QMessageBox, QPushButton, QScrollBar,
-    QStyle, QTableView, QTableWidget, QToolButton, QVBoxLayout, QWidget)
+    QApplication, QDialog, QFrame, QGridLayout, QHBoxLayout, QHeaderView,
+    QInputDialog, QItemDelegate, QLabel, QLineEdit, QMessageBox, QPushButton,
+    QScrollBar, QStyle, QTableView, QTableWidget, QToolButton, QVBoxLayout,
+    QWidget)
 from spyder_kernels.utils.lazymodules import numpy as np, pandas as pd
+if TYPE_CHECKING:
+    from matplotlib.figure import Figure
```

Comment:
```suggestion
```

Let's group all `TYPE_CHECKING` imports after all regular imports below (I've seen that's what other projects do).